test/core/fixtures/transformation/es6.spec.template-literals/only/expected.js

```diff
@@ -1,3 +1,3 @@
 "use strict";
 
-var foo = "" + String(test);
+var foo = String(test);
```

Comment:
Actually it seems kind of weird the the actual is also  ```var foo = `${test}` ``` but this one casts to a string? @Dashed